src/Community/data.json

```diff
@@ -0,0 +1,78 @@
+{
+  "hero": {
+    "picture": "/static/img/community/hero.png",
+    "url": "http://event.link"
+  },
+  "section": {
+    "contribute": {
+      "title": "Contribute",
+      "description": "DVC is 100% open-source and we welcome contributions of all sorts to our code and docs."
+    },
+    "events": {
+      "title": "Events",
+      "description": "Conferences, meetups and talks to learn more about DVC or meet in person."
+    },
+    "learn": {
+      "title": "Learn",
+      "description": "Every month we are sharing here our news, findings, interesting reads, community takeaways, and everything along the way."
+    },
+    "meet": {
+      "title": "Meet the Community",
+      "description": "Ask questions, find answers, share your ideas and connect with DVC community."
+    }
+  },
+  "documentation": [
+    {
+      "url": "/doc/get-started",
+      "title": "Get started",
+      "description": "Learn how to set up your project"
+    },
+    {
+      "url": "/doc/tutorials",
+      "title": "Tutorials",
+      "description": "Learn how to solve most typical tasks"
+    },
+    {
+      "url": "/doc/changelog",
+      "title": "Changelog",
+      "description": "See what's new in DVC core "
+    }
+  ],
+  "userContent": [
+    {
+      "url": "/1",
+      "title": "How we use DVC in our project",
+      "author": "James Hopfield",
+      "date": 1576072266986
+    },
```

Comment:
Can we use human readable dates instead of timestamps for these? e.g. `yyyy-mm-dd` format

Reply:
We can use any format there, that can be parsed by `new Date()`, so `yyyy-mm-dd` should work too.

Reply:
Great, so do you want to update this to the corresponding date in readable format? 🙂

Reply:
It's just a placeholder data now, I think to replace it then we will add actual events.